middlewares/validators/validator.helper.js

```diff
@@ -326,7 +326,7 @@ function applicationValidator(fieldLocation, fieldname, optional = true) {
             hasValid.linkedIn = (!app.portfolioURL.linkedIn || typeof (app.portfolioURL.linkedIn) === "string");
             hasValid.other = (!app.portfolioURL.other || typeof (app.portfolioURL.other) === "string");
             hasValid.jobInterest = (!app.jobInterest || jobInterests.includes(app.jobInterest));
-            hasValid.skills = (!app.skills || isMongoIdArray(app.skills));
+            hasValid.skills = (!app.skills || Array.isArray(app.skills));
```

Comment:
Perhaps consider using alphaArrayValidator. It checks that each element in the array is a string, which fits the model better.

Reply:
done